### PR TITLE
Go solutions

### PR DESCRIPTION
**Pull request by evbogdanov**

### CODE REVIEW

**Comment by sergeyklay** [comment]
Awesome!

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*.DS_Store
```
Comment (sergeyklay) [inline .gitignore]: Please do not use this ignore file to define platform specific files.

For these purposes create a global `.gitignore` file, which is a list of rules
for ignoring files in every Git repository on your computer.

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

**Review by sergeyklay — CHANGES_REQUESTED** [review]

**Comment by sergeyklay** [comment]
@evbogdanov Looks like this repo is abandoned. I've created a fork and merged your Go solutions: https://github.com/sergeyklay/programming_skills_test

**Comment by evbogdanov** [comment]
@sergeyklay Awesome, keep up the good work!

**Pull request merged**